### DISCUSSION TITLE
Problema compartiendo archivos

### DISCUSSION THREAD

**Discussion by Mauriceac** [Cotalker/documentation#560] [4. Bug Reports]
Videos en formato ***.mov** (y quizás otros formatos también) no son enviados a través de la interfaz gráfica (error: _“Document extension not allowed”_), pero pueden ser subidos por API request: `POST media/file/upload`. 

Para compartir el video como mensaje en un canal, `messages.contentType` tiene que estar puesto en `video/mp4` (y no `video/quicktime`).

**Comment by acedward** [comment]
Por ahora *.mov no son soportados,
principalmente porque no podemos convertirlos fácilmente en nuestro backend
(transformamos todos los videos a formatos visibles en todos los dispositivos y que  reduciendo su peso)
